app/src/main/java/chat/rocket/android/settings/ui/SettingsFragment.kt

```diff
@@ -1,6 +1,8 @@
 package chat.rocket.android.settings.ui
 
+import android.content.ActivityNotFoundException
```

Comment:
Where is this import being used (the same for the `android.net.Uri`) ? 🤔

Reply:
Oh my bad, forgot to remove